target/wddr101/include/drivers/ddr_dfich_csr.h

```diff
@@ -0,0 +1,808 @@
+
```

Comment:
Will this file take precedent over ddr_dfich_csr.h used in the wddr100 folder?

Reply:
I'm not trying to be snarky but I think this is a good learning opportunity. Please look into how include headers are found by the compiler, how they are specified in cmake and what that means in terms of the compilation command. If you look into those, then you will find the answer. It will explain how the target directories can override / extend functionality via headers.